### ISSUE TITLE
why the input and outpu have not the rain data?

### ISSUE THREAD

**Issue by longtsing** [198808xc/Pangu-Weather#24]
For chinese ,the rain data is more important than other data especialy now. Why the input and output have not the rain data? Maybe there is some reason about it ?
And I use the PanGu-weather model in CMARA (the CMA reanalysis data) ,I find the temperature in 2m is serious underestimation. For example , the T2M that forecasted by PanGu-weather using the CMARA Data and ERA5( in the mid china mainland)  is more low than the nature temperature  8 degree.

**Comment by 198808xc** [comment]
1. When we started the project, we thought that the precipitation (raining) contains lots of randomness and is hard to predict, so we simply removed this variable from the training data. We noticed that a researcher tested Pangu in precipitation and released a brief report (in Chinese): https://mp.weixin.qq.com/s/LSe5HWkUttY98ceGD8nCnQ. Maybe you can take it as a reference.

2. I am not sure what happened. 8 degrees is not a small amount, so I guess some bugs exist. Pangu is not very sensitive to the initial values, as many researchers have tested, e.g. the running operational system at ECMWF is directly testing Pangu on their IFS initial data which is different from the ERA data. Also, Pangu was tested on GRAPES data -- even with such data, they did not observe such a big error. Please refer to the technical report from ECMWF: https://arxiv.org/abs/2307.10128.

**Comment by liwenyu0625** [comment]
> 1. When we started the project, we thought that the precipitation (raining) contains lots of randomness and is hard to predict, so we simply removed this variable from the training data. We noticed that a researcher tested Pangu in precipitation and released a brief report (in Chinese): https://mp.weixin.qq.com/s/LSe5HWkUttY98ceGD8nCnQ. Maybe you can take it as a reference.
> 2. I am not sure what happened. 8 degrees is not a small amount, so I guess some bugs exist. Pangu is not very sensitive to the initial values, as many researchers have tested, e.g. the running operational system at ECMWF is directly testing Pangu on their IFS initial data which is different from the ERA data. Also, Pangu was tested on GRAPES data -- even with such data, they did not observe such a big error. Please refer to the technical report from ECMWF: https://arxiv.org/abs/2307.10128.

I would like to ask if Pangu will follow up with a rainfall forecasting product, which is important for our hydrology profession. I want to use Pangu to build a meteorological and hydrological hydrodynamic model, but I find that it lacks important rainfall field data.

**Comment by bmkor** [comment]
> > 1. When we started the project, we thought that the precipitation (raining) contains lots of randomness and is hard to predict, so we simply removed this variable from the training data. We noticed that a researcher tested Pangu in precipitation and released a brief report (in Chinese): https://mp.weixin.qq.com/s/LSe5HWkUttY98ceGD8nCnQ. Maybe you can take it as a reference.
> > 2. I am not sure what happened. 8 degrees is not a small amount, so I guess some bugs exist. Pangu is not very sensitive to the initial values, as many researchers have tested, e.g. the running operational system at ECMWF is directly testing Pangu on their IFS initial data which is different from the ERA data. Also, Pangu was tested on GRAPES data -- even with such data, they did not observe such a big error. Please refer to the technical report from ECMWF: https://arxiv.org/abs/2307.10128.
> 
> I would like to ask if Pangu will follow up with a rainfall forecasting product, which is important for our hydrology profession. I want to use Pangu to build a meteorological and hydrological hydrodynamic model, but I find that it lacks important rainfall field data.

Yes. I echo what @liwenyu0625 suggests. Know that there will be considerations but we believe that such a release or follow up would benefit many many parties. Many thanks for this great work.
